Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -112,51 +112,62 @@ def signoff!(path, pr: nil, dry_run: false)
     end
   end
 
-  def determine_bump_subject(old_contents, new_contents, formula_path, reason: nil)
-    formula_path = Pathname(formula_path)
-    formula_name = formula_path.basename.to_s.chomp(".rb")
+  def determine_bump_subject(old_contents, new_contents, subject_path, reason: nil)
+    subject_path = Pathname(subject_path)
+    subject_name = subject_path.basename.to_s.chomp(".rb")
+    is_cask      = subject_path.to_s.include? "Casks"
 
-    new_formula = begin
-      Formulary.from_contents(formula_name, formula_path, new_contents, :stable)
+    new_package = begin
+      Formulary.from_contents(subject_name, subject_path, new_contents, :stable)
     rescue FormulaUnavailableError
-      nil
+      nil unless is_cask
+      begin
+        Cask::CaskLoader.load(new_contents.dup, config: Cask::Config.defaults)
+      rescue Cask::CaskUnavailableError
+        nil
+      end
     end
 
-    return "#{formula_name}: delete #{reason}".strip if new_formula.blank?
+    return "#{subject_name}: delete #{reason}".strip if new_package.blank?
 
-    old_formula = begin
-      Formulary.from_contents(formula_name, formula_path, old_contents, :stable)
+    old_package = begin
+      Formulary.from_contents(subject_name, subject_path, old_contents, :stable)
     rescue FormulaUnavailableError
-      nil
+      nil unless is_cask
+      begin
+        Cask::CaskLoader.load(old_contents.dup, config: Cask::Config.defaults)
+      rescue Cask::CaskUnavailableError
+        nil
+      end
     end
 
-    return "#{formula_name} #{new_formula.stable.version} (new formula)" if old_formula.blank?
+    return "#{subject_name} #{new_package.version} (new #{is_cask ? "cask" : "formula"})" if old_package.blank?
```

Comment:
The `is_cask` would be better as a separate variable rather than inline I think.